iothub/device/src/Transport/Amqp/PendingAmqpTwinOperation.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.Amqp;
+
+namespace Microsoft.Azure.Devices.Client.Transport.Amqp
```

Comment:
Remind me, did we want all classes in this library to just use the ```Microsoft.Azure.Devices.Client``` namespace?

Reply:
I think for transport specific stuff we decided it was best to isolate them by namespace.